### PR TITLE
Merging latest master

### PR DESCRIPTION
**Pull request by langfzac**
Needed for up-to-date init_nbody methods

### CODE REVIEW

File: test/test_ttv_elements.jl [hunk 1]
```diff
@@ -27,7 +27,7 @@ ntt = zeros(Int64,n)
 # Make an array, tt,  to hold transit times:
 # First, though, make sure it is large enough:
 for i=2:n
-  ntt[i] = ceil(Int64,tmax/elements[i,2])+3
+  global ntt[i] = ceil(Int64,tmax/elements[i,2])+3
```
Comment (ericagol) [inline test/test_ttv_elements.jl:30]: Why the global before ntt?

File: test/test_ttv_elements.jl [hunk 2]
```diff
@@ -57,7 +57,7 @@ for jq=1:n_body
       for k=1:count2[i]
         # Ignore inclination & longitude of nodes variations:
         if iq != 5 && iq != 6 && ~(jq == 1 && iq < 7) && ~(jq == i && iq == 7)
-          mask[i,k,iq,jq] = true
+         global mask[i,k,iq,jq] = true
```
Comment (ericagol) [inline test/test_ttv_elements.jl:60]: Why the global before mask?

**Review by ericagol — COMMENTED** [review]
These changes look good, and the tests pass.  I'm curious why you put a "global" in front of ntt & mask?

**Comment by langfzac** [comment]
When I was using Julia v0.7, to update the code for v1.1, the warning messages were saying the preferred style is to write a "global" in front of any variable that is not instantiated inside the for loop. I assume it's because any variable that's declared inside the for loop is part of the local scope of the loop. This style avoids ambiguity and makes it a little more readable. I found out later that it's not required, so it's not included in other parts of the code (mainly because I forgot since there's no errors thrown). The warning is 'implicit assignment to global variable' if you look at the v0.7 build on the Travis-CI page.

**Comment by ericagol** [comment]
Thanks, that makes sense!

**Pull request merged**